pkg/pheanstalk/PheanstalkConsumer.php

```diff
@@ -97,6 +99,13 @@ public function reject(Message $message, bool $requeue = false): void
         $this->acknowledge($message);
     }
 
+    private function checkMessageHasJob(PheanstalkMessage $message, string $verb)
+    {
+        if (false == $message->getJob()) {
+            throw new \LogicException("The message could not be $verb because it does not have job set.");
+        }
```

Comment:
Could you please revert this? I prefer to keep this separately

Reply:
yes. please

Reply:
Done.